docs/sources/panels/visualizations/time-series/_index.md

```diff
@@ -0,0 +1,79 @@
++++
+title = "Time series panel"
+keywords = ["grafana", "graph panel", "time series panel", "documentation", "guide", "graph"]
+weight = 1200
++++
+
+# Time series panel
+
+> **Note:** This is a beta feature. Time series panel is going to replace the Graph panel in the future releases.
+
+Time series panel is a robust visualization to plot time series data. It can render as a line, a path of dots, or a series of bars. This type of graph is versatile enough to display almost any time-series data.
+
+For Time series panel examples, refer to the Grafana Play dashboard [New Features in v7.4](https://play.grafana.org/d/nP8rcffGk/new-features-in-v7-4?orgId=1).
+
+## Data and field options
+
+Time series visualizations allow you to apply:
+
+- [Data transformations]({{< relref "../../transformations/_index.md" >}})
+- [Field options and overrides]({{< relref "../../field-options/_index.md" >}})
+- [Thresholds]({{< relref "../../thresholds.md" >}})
+
+You can also use field options to create different types of graphs or adjust your axes:
+
+- [Create line graphs]({{< relref "create-line-graphs.md" >}})
+- [Create bar graphs]({{< relref "create-bar-graphs.md" >}})
+- [Create scatter plot graphs]({{< relref "create-scatter-plot-graphs.md" >}})
```

Comment:
we might want to avoid calling these things scatter, since the new X/Y chart will be the actual scatter.

scatter plots typically have a numeric (non-time) x axis and can have multiple y datapoints for each x, which is not the case for time series.

Reply:
Like the above, this is how to show time series in a scatter plot chart.

According to both my research and my SME who has spent 15 years in analytics and knows a few things about visualizations, scatter point charts can and often do chart time-series data. They don't have to, and we will be coming up with a better tool for that soon, but I think this is good (and valid) for now.

I am open to better names if you have any suggestions, just keep in mind that we want users to know what we are talking about when they see the descriptions.

Reply:
ok, sorry for the suggestion. https://www.google.com/search?q=scatter+point+chart&source=lnms&tbm=isch

Reply:
I appreciate suggestions! I just don't know what else to call these topics, and the graph sure looks like a scatter plot.

Bar line graph instead of bar chart?
Points line graph instead of scatter plot?

Any ideas?

Reply:
maybe series lines, series points and series bars. or line series, bar series, point series?

naming things sucks :(

Reply:
Yea, let's not call this scatter plots, please. We will have a separate visualization focused on that.

Reply:
In regards to calling the topic "Create scatter plot," we can have more than one away of making a particular type of graph.

Also, the official name of the visualization is apparently going to be XY plot or something similar, so that reduces the confusion risk. 

Either way, that's a problem for after 7.4. Right now, we want to try to give the users the best documentation for 7.4 that we can get.

In regards to if this is a scatter plot or not, is there averaging or summing going on behind the scenes to get the points or are we presenting all the data points as-is in the visualization?

Reply:
Data is presented as is -- BUT it must be sorted time series data.

While time series rendered as points may technically fall into a scatter plot category -- saying we support scatter plots will greatly disappoint many people, since this is an extremely limited view of what scatter plots do.  Note there are ~5 flavors of scatter plots in the plugin directory because the graph and timeseries panel do not really support what people need from scatter plots.

Similar with "Bar chart" -- we are adding a "Bar chart" panel because it does what people think bar charts do, not time series rendered as bars.

Reply:
How about -

Graph time series as lines
Graph time series as bars
Graph time series as points

Would that be accurate and acceptable?

Reply:
These topics are more like tasks, and we generally want to start tasks with imperative verbs. Create, graph, adjust, and so on.

Reply:
File names updated. Please take a look.